code/modules/projectiles/guns/ego_gun/waw.dm

```diff
@@ -0,0 +1,10 @@
+/obj/item/gun/ego_gun/hornet
+	name = "hornet"
+	desc = "The kingdom needed to stay prosperous, and more bees were required for that task.\
+	The projectiles relive the legacy of the kingdom as they travel toward the target."
+	icon_state = "hornet"
+	inhand_icon_state = "hornet"
+	w_class = WEIGHT_CLASS_NORMAL
+	ammo_type = /obj/item/ammo_casing/caseless/ego_hornet
```

Comment:
Make it unable to be used with one hand like the fourth match is, also make it bulky (can still fit on your belt